homeassistant/components/vicare/strings.json

```
@@ -362,6 +362,12 @@
           "production": "Producing"
         }
       },
+      "room_temperature": {
+        "name": "Temperature"
+      },
+      "room_humidity": {
+        "name": "Humidity"
```

Should this be a reference instead? 
```suggestion
        "name": "[%key:component::sensor::temperature%]"
      },
      "room_humidity": {
        "name": "[%key:component::sensor::humidity%]"
```